### PR TITLE
MySQL Installer Updates

### PR DESCRIPTION
**Pull request by tallona**
Updated some keywords to use standard words for easier reading, added 'idx_' prefix for all indexes, removed NOT NULL from primary key columns as it is implicitly added when a column is defined as a primary key.

Signed-off-by: Adam Tallon adamtallon@gmail.com

### CODE REVIEW

**Comment by live627** [comment]
Neat.

File: other/install_2-1_mysql.sql [hunk 1]
```diff
@@ -1825,12 +1825,12 @@ INSERT INTO {$db_prefix}scheduled_tasks
 	(id_task, next_time, time_offset, time_regularity, time_unit, disabled, task, callable)
 VALUES
 	(1, 0, 0, 2, 'h', 0, 'approval_notification', ''),
-	(3, 0, 60, 1, 'd', 0, 'daily_maintenance', ''),
+	(3, 0, 60, 1, 'd', 0, 'daily_maINTenance', ''),
```
Comment (live627) [inline other/install_2-1_mysql.sql]: This will break SMF. Please revert.

**Comment by tallona** [comment]
Thanks, added that change.

**Comment by live627** [comment]
There are still a bajillion inserts that have been modified to use the wrong case. Can you also revert those?

**Comment by tallona** [comment]
Sorry about that, find and replace can be your friend but can also be your enemy at the best of times :)

**Pull request merged**